api/cpp/include/slint-testing.h

```diff
@@ -11,6 +11,9 @@
 #    ifdef SLINT_FEATURE_EXPERIMENTAL
 
 namespace slint::testing {
+
+using slint::cbindgen_private::AccessibleRole;
```

Comment:
Actually, the problem with that is that there will not be a C++ documentation for AccessibleRole AFAIK

Reply:
There are no docs for experimental features right now at all. The entire API is undocumented. That's another issue... yeah